### PR TITLE
Add existing pages.18f.gov redirects from federalist-redirects

### LINKED ISSUE #17: Include the rest of federalist-pages stuff for pages/guides.18f.gov
I think we need all of https://github.com/18F/federalist-redirects/blob/master/nginx.conf#L84-L90 to also be included in our nginx config since this app will completely take over handling of pages.18f.gov/guides.18f.gov.

### PR DESCRIPTION
**Pull request by jseppi**
Closes #17

### CODE REVIEW

File: pages.yml [hunk 1]
```diff
@@ -4,3 +4,5 @@
 # Add redirects that have a change with
 # - from: oldname
 #   to: newname
+- from: state-faq
```
Comment (jseppi) [inline pages.yml]: This config takes the place of https://github.com/18F/federalist-redirects/blob/master/nginx.conf#L84

File: templates/nginx.conf.nj [hunk 2]
```diff
@@ -41,7 +41,16 @@ http {
     rewrite ^/{{ page.from }}(.*)$ https://{{ page.to }}.18f.gov$1;
     {% endfor %}
 
+    rewrite ^/intake/.* https://18f.gsa.gov/contact/ redirect;
+    rewrite ^/designstandards/.* https://standards.usa.gov/ redirect;
+    rewrite ^/pages/.* https://github.com/18F/pages/ redirect;
```
Comment (jseppi) [inline templates/nginx.conf.nj]: @wslack This one seems weird to me. Basically it redirects https://pages.18f.gov/pages/whatever` back to the pages GitHub repo. Is it necessary to keep?

Copied from https://github.com/18F/federalist-redirects/blob/master/nginx.conf#L87

File: pages.yml (hunk 1 above)
Comment (wslack) [inline pages.yml]: 👍

File: templates/nginx.conf.nj (hunk 2 above)
Comment (wslack) [inline templates/nginx.conf.nj]: I don't think so, though we're want to keep sending /pages/whatever to pages.18f.gov

**Comment by wslack** [comment]
This broadly looks good. Can you add a comment noting that the proxy-pass will be going away ASAP (after sites are all live) so its clear that line is a short term solution?

**Pull request merged**